### PR TITLE
Fix Unp tmp dir with capital letters

### PR DESCRIPTION
**Pull request by rafatower**
The problem is that the `.downcase` call through `normalize->underscore`is downcasing the full path, including the directory and the file name.

Later on, it tries to retrieve the file from a different path, and the file is not there.

This fixes it by normalizing just the filename (and not the full path), which is then expected to not have capitals or especial characters in some other parts of the importer (table names are taken from file names).

### CODE REVIEW

**Comment by rafatower** [comment]
The error from the test before the fix:
```
17:32:43 rspec ./services/importer/spec/unit/unp_spec.rb:29 # CartoDB::Importer2::Unp#run lets an uncompressed file pass through and store it in a tmp dir with uppercase chars
17:32:43 
17:32:43 Failures:
17:32:43 
17:32:43   1) CartoDB::Importer2::Unp#run lets an uncompressed file pass through and store it in a tmp dir with uppercase chars
17:32:43      Failure/Error: unp.run(uncompressed_file)
17:32:43      Errno::ENOENT:
17:32:43        No such file or directory @ rb_file_s_rename - (/tmp/IMPORTS/20190221-3797-ktgujy/duplicated_column_name.csv, /tmp/imports/20190221-3797-ktgujy/duplicated_column_name.csv)
17:32:43      # ./services/importer/lib/importer/unp.rb:184:in `rename'
17:32:43      # ./services/importer/lib/importer/unp.rb:184:in `rename'
17:32:43      # ./services/importer/lib/importer/unp.rb:165:in `normalize'
17:32:43      # ./services/importer/lib/importer/unp.rb:63:in `without_unpacking'
17:32:43      # ./services/importer/lib/importer/unp.rb:51:in `run'
17:32:43      # ./services/importer/spec/unit/unp_spec.rb:33:in `block (3 levels) in <top (required)>'
17:32:43 
17:32:43 Finished in 4 minutes 45.71 seconds
```

**Comment by rafatower** [comment]
@jgoizueta adding you to review as chances are you'll have to deploy it soon.

**Comment by rafatower** [comment]
Smoke tested in staging :heavy_check_mark:

File: services/importer/spec/unit/unp_spec.rb [hunk 1]
```diff
@@ -310,7 +348,7 @@
     end
   end
 
-  def zipfile_factory(dir = '/var/tmp/bogus', filename: 'bogus.zip')
+  def file_factory(dir = '/var/tmp/bogus', filename: 'bogus.zip')
     zipfile = "#{dir}/#{filename}"
```
Comment (jgoizueta) [inline services/importer/spec/unit/unp_spec.rb]: This is just nit-picking (on code untouched by the PR) but should be `zipfile = File.join(dir, filename)`

File: services/importer/spec/unit/unp_spec.rb [hunk 2]
```diff
@@ -18,16 +18,54 @@
...
+      FileUtils.rm_rf(unp.temporary_directory)
+    end
+
+    it 'can deal with filenames with capital letters (compressed file)' do
+      zip_file = file_factory(filename: 'TestShape.zip')
+      unp = Unp.new
+
+      unp.run(zip_file)
+
+      # NOTE: the input zip file contains file names capitalized, e.g:
```
Comment (jgoizueta) [inline services/importer/spec/unit/unp_spec.rb]: nice

**Pull request merged**